tests/unit/aws/test_access.py

```diff
@@ -864,3 +864,68 @@ def command_call(_: str):
     assert len(roles) == 1
     assert len(roles[0].paths) == 2
     external_locations.snapshot.assert_called_once()
+
+
+def test_delete_uc_roles(mock_ws, installation_multiple_roles, backend, locations):
+    aws = create_autospec(AWSResources)
+    aws.validate_connection.return_value = {}
+    aws.delete_role.return_value = []
+    aws_resource_permissions = AWSResourcePermissions(installation_multiple_roles, mock_ws, aws, locations)
+    mock_ws.storage_credentials.list.return_value = [
+        StorageCredentialInfo(
+            id="1",
+            name="cred1",
+            aws_iam_role=AwsIamRoleResponse("arn:aws:iam::12345:role/uc-role1"),
+        )
+    ]
+    role_creation = IamRoleCreation(installation_multiple_roles, mock_ws, aws_resource_permissions)
+    prompts = MockPrompts({"Select the list of roles *": "1", "The above storage credential will be impacted *": "Yes"})
+    role_creation.delete_uc_roles(prompts)
+    assert aws.delete_role.assert_called
```

Comment:
```suggestion
    aws.delete_role.assert_called()
```
`assert_called` is a method that always exists, hence it's "truthy", so `assert aws.delete_role.assert_called` will always pass.

can you also assert the arguments it's called with?